src/Compilers/Core/Portable/InternalUtilities/ConsList`1.cs

```diff
@@ -142,7 +150,7 @@ public override string ToString()
                     result.Append(", ");
                 }
 
-                result.Append(list._head);
+                result.Append(list._head!);
```

Comment:
And here

Reply:
For this one I just switched to `list.Head`. 😄

Reply:
➡️ Fixed